kernel/src/kernel.c

```diff
@@ -39,6 +39,9 @@ void kernel_main(uint32_t mbaddr, uint32_t mbmagic,
 		qemu_write_string("m: 0x%x -> 0x%x\n", module, *module);
 	}
 	//module_entry_point();
+	srand(42);
+	int random = rand(5, 1452);
+	printf("%d", random);
```

Comment:
we have a optional.c file that stores options and minor tests in src/ and we are calling optional inits in kernel.c instead of calling them direcly on main functin, we can create something like this: 
in options.h:
`#define TEST_RANDOM TRUE`
in options.c:
inside the void optional_inits() function:
```
#if TEST_RANDOM
       srand(42);
	int random = rand(5, 1452);
	qemu_write_strinf("%d", random);
#endif
```
and then the kernel_main optional_inits() and that would run